lib/dbservice_web/views/session_occurence_view.ex

```diff
@@ -21,17 +19,4 @@ defmodule DbserviceWeb.SessionOccurenceView do
       session_id: session_occurence.session_id
     }
   end
```

Comment:
Why you removed this piece of code? We need to show session_occurence_with_users no?